gas/config/tc-riscv.c

```diff
@@ -988,12 +991,17 @@ validate_riscv_insn (const struct riscv_opcode *opc, int length)
       case 'b':
 	if (*p == '1')
 	  {
-	    used_bits |= ENCODE_ITYPE_IMM(-1U); /* For loop I type pc rel displacement */
+	    used_bits |= ENCODE_ITYPE_IMM(-1U);
 	    ++p; break;
 	  }
 	else if (*p == '2')
 	  {
-	    used_bits |= ENCODE_CV_HWLP_UIMM5(-1U); /* For loop I1 type pc rel displacement */
+	    used_bits |= ENCODE_CV_HWLP_UIMM5(-1U);
```

Comment:
Same here - why remove the comment?

Reply:
@jeremybennett The comments relate to I1TYPE, I3TYPE etc. which made no sense. Mary renamed them in the previous commit but we forgot to take them out here